### ISSUE TITLE
cant find race-free state for 5004b083e94e008e9f88bdb9ede5a724 (error 236)

### ISSUE THREAD

**Issue by yoza111** [keybase/client#24715]
Hi, my whole team on Keybase has gone down. I get the following message, error recalculating name for shalimardairy: cant find race free state 5004b083e94e008e9f88bdb9ede5a724 (error 236). this will pop up when im trying to post a picture or write a message in any of my teams channels. can dm me for username or any other details that are required. Thanks!

**Comment by hoffmang9** [comment]
Adding that multiple folks on the chia_network team have seen this problem too.

**Comment by AmorImpermissus** [comment]
Have the exact same issue, started on Windows 10, and in my case, it happened after I tried to create a team. It generated an almost identical error (except for the uid associated with my user instead, 95ee15759d9b44fad5e9a33f27540b19) in my case at the 'channel creation' step. I ended up needing to hit 'back' until the team creation dialog was gone. Even though it gave an error, it still gave me the team as an option to the left. But I wasn't able to create any additional channels in the team. I didn't add any other users to my team, had planned to do that later. I also tried to upload a picture as the team profile picture and this spun for an inordinately long time and then gave an error that the image wasn't supported (which is odd since I was able to use literally the exact same file 2 days ago on another team). Thinking it was related to perhaps some cache or local file corruption, I completely signed out of keybase on my Windows system, only to be greeted with the error 236 message right after I give the app my username. I also tried completely blowing away all the files on my Windows computer.

I was unable to see any devices in the list on my Android phone, but the account was still signed in and I was able to send/receive messages with other keybase users. I cleared the app data on my android app to see if the same error 236 showed up when I tried to sign back in on that device (yes, it does).

I have also attempted to sign back in using the cli on Linux, no joy there either. I still have a paper key so I'm sure I can sign back in as needed, and my account on keybase.io still shows all the history/devices.

I have attached the keybase log files from %LOCALAPPDATA% on my Win10 system. Hopefully nothing sensitive there lol
[Keybase.zip](https://github.com/keybase/client/files/7638260/Keybase.zip)
